node/src/components/consensus/highway_core/state.rs

```diff
@@ -735,15 +747,15 @@ impl<C: Context> State<C> {
     }
 
     /// Returns an iterator over units (with hashes) by the same creator, in reverse chronological
-    /// order, starting with the specified unit.
+    /// order, starting with the specified unit. Returns an empty iterator if the unit isn't known.
```

Comment:
Why? All units should be known before calling it. If the unit is not found then we have a bug.

Reply:
It's just another hidden `unwrap` that I thought would be easy to remove. But if you prefer I can go back to panicking, and document the panic.

Reply:
Yes, I think so.

Reply:
Done in https://github.com/CasperLabs/casper-node/pull/622/commits/4d3cd9216258640c30c464fcb494cc11cb0992be.